### PR TITLE
Updates to prep for tagging releases

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: src/mozanalysis/bootstrap.py [hunk 1]
```diff
@@ -52,7 +53,7 @@ def bootstrap(sc, data, stat_fn, num_iterations=2000, confidence_level=0.95):
 
     broadcast_data.unpersist()
 
-    p = _percentile(data, confidence_level)
+    p = _percentile(stats, confidence_level)
```
Comment (jezdez) [inline src/mozanalysis/bootstrap.py]: That seems unrelated?

Comment (robhudson) [reply src/mozanalysis/bootstrap.py]: Yes, it's now in another PR and this branch will be rebased on that when it is merged. You have eagle eyes. :eagle: :eyes:
